ADALiOS/ADALiOS/ADKeychainTokenCacheStore.m

```diff
@@ -536,6 +554,15 @@ -(void) removeAllForUser: (NSString*) userId
                    error: (ADAuthenticationError* __autoreleasing*) error
 {
     API_ENTRY;
+    if (mSharedCache)
+    {
+        AD_LOG_ERROR(@"******** removeAllWithError is being called. This will remove all tokens from the ADAL shared cache that your application has access to. ********", AD_ERROR_UNEXPECTED, nil);
```

Comment:
Shouldn't the message be "removeAllForUser is being called"? :P

Shall we add the similar logging to method [removeAllForUser: error:] below? :P